tuning/scripts/provision_verification.sh

```diff
@@ -109,20 +110,17 @@ cp ${VERIFY_LIBRARY_ARCHIVE}/* ${ARCHIVE_PATH}
 cp ${ARCHIVE_PATH}/*yaml ${ASM_PATH}
 
 MERGE_SCRIPT=${TENSILE_PATH}/Tensile/Utilities/merge_rocblas_yaml_files.py
-MESSAGE_SCRIPT=../archive/massage.py
+#MESSAGE_SCRIPT=../archive/massage.py
+MESSAGE_SCRIPT=${VERIFY_LIBRARY_ARCHIVE}/massage.py
 
 EXE_MERGE="python ${MERGE_SCRIPT} ${ASM_PATH} ${EXACT_PATH} ${MERGE_PATH}"
 ${EXE_MERGE}
 
 cp ${MERGE_PATH}/* ${VERIFY_LIBRARY_ASM}
-cp ${MERGE_PATH}/vega20*{SB,DB}* ${ARCHIVE_PATH}
-cp ${ARCHIVE_PATH}/*yaml ${VERIFY_LIBRARY_ASM}
-cp ${ARCHIVE_PATH}/*yaml ${VERIFY_LIBRARY_ARCHIVE}
-
-pushd ${VERIFY_LIBRARY_ASM} > /dev/null
-python ${MESSAGE_SCRIPT}
-popd > /dev/null
+cp ${MERGE_PATH}/* ${VERIFY_LIBRARY_ARCHIVE}
+cp ${MERGE_PATH}/vega20*{SB,DB}* ${MASSAGE_PATH}
```

Comment:
shouldn't HB included in this list?

Reply:
Currently there are not hgemm logic files in the archive and hence we are not massaging those.